### PR TITLE
Update auth.adoc

### PR DESCRIPTION
**Pull request by jodydadescott**
Add pc-token sub command documentation

### CODE REVIEW

**Comment by iansk** [comment]
@rcaballeromx Can you please review these changes and let me know if they're ok to merge as-is?

File: cns/saas/admin_guide/apoctl/auth.adoc [hunk 1]
```diff
@@ -148,6 +148,16 @@ For instance:
 
  apoctl auth google --open-with 'Google Chrome'
 
+
+=== `+pc-token+` subcommand
+
+The `+pc-token+` subcommand allows to retrieve a Microsegmentation token using Prisma
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/auth.adoc:154]:
```suggestion
The `+pc-token+` subcommand allows you to retrieve a microsegmentation token using Prisma
```

File: cns/saas/admin_guide/apoctl/auth.adoc [hunk 2]
```diff
@@ -148,6 +148,16 @@ For instance:
...
+tokens.
+
+Example:
+
+ apoctl auth pc-token --token "$TOKEN"
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/auth.adoc:159]: @iansk The examples on this page do not have markup making them code blocks. Is that intentional?

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
Missing a pronoun for clarity.

**Comment by Pubs-MV** [comment]
closing this request as the changes from PR #168 and #65 are in #230

**Pull request closed without merging**